pnpm-lock.yaml

```diff
@@ -245,7 +245,7 @@ importers:
       '@reactioncommerce/logger': link:../../packages/logger
       '@reactioncommerce/nodemailer': 5.0.5
       '@reactioncommerce/random': link:../../packages/random
-      '@snyk/protect': 1.1032.0
```

Comment:
You should remove this (Snyk), it's going to just keep causing problems

Reply:
removed